montreal-python/videos/00000.json

```diff
@@ -0,0 +1,23 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "",
+  "duration": 59,
+  "language": "eng",
+  "recorded": "2011-12-22",
+  "related_urls": [
+    {
+      "label": "group web",
+      "url": "https://montrealpython.org"
+    }
+  ],
+  "speakers": [],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/tPDhu-PacfU/maxresdefault.jpg",
+  "title": "00000",
```

Comment:
Since we've never add the proper title i'm guessing we will be able to update this in a subsequent PR ?